hybrid/HybridSampleApps/NoteSync/build.gradle

```diff
@@ -5,8 +5,9 @@ dependencies {
 }
 
 android {
-  compileSdkVersion 21
-  buildToolsVersion "21.1.1"
+  compileSdkVersion 23
+  buildToolsVersion '23.0.1'
+  useLibrary  'org.apache.http.legacy'
```

Comment:
ContentSoqlSyncDownTarget in the NoteSync project makes use of org.apache.http.entity.StringEntity, otherwise you are right, it wouldn't need the useLibrary.